test/unit/uk/gov/hmrc/apinotificationpull/controllers/NotificationsControllerSpec.scala

```diff
@@ -18,18 +18,24 @@ package uk.gov.hmrc.apinotificationpull.controllers
 
 import java.util.UUID
 
-import play.api.test.FakeRequest
+import org.scalatest.mockito.MockitoSugar
+import play.api.http.HeaderNames._
 import play.api.http.Status._
+import play.api.test.FakeRequest
 import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
 
-class NotificationsControllerSpec extends UnitSpec with WithFakeApplication {
+class NotificationsControllerSpec extends UnitSpec with WithFakeApplication with MockitoSugar {
   "delete notification by id" when {
+    val headerValidator = new SuccessfulHeaderValidator
```

Comment:
fields should be private ie private `private val headerValidator = new SuccessfulHeaderValidator`